app/Models/Post.php

```diff
@@ -22,7 +22,7 @@ class Post extends Model
      * Always load a user's own reaction,
      * if they're logged-in.
      */
-    protected $with = ['reaction', 'tags'];
+    // protected $with = ['reaction', 'tags'];
```

Comment:
Let's add these back in!

Reply:
ah yes you beat me to that! Just updated that :)